packages/generate-scaffold/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as c from "ansi-colors";
+import chalk from "chalk";
+import { exec } from "child_process";
+import * as fs from "fs";
+import * as fse from "fs-extra";
+import * as inquirer from "inquirer";
+import { resolve } from "path";
+
+const cl = console;
+const log = cl.log;
+
+const USER_DIRECTORY: string = process.env.PWD ? process.env.PWD : process.cwd();
+import * as QUESTIONS from "./utils/questions";
+const resolveProjectDirectory = (path: string, scaffoldProjectName: string): string =>
+resolve(`${USER_DIRECTORY}/${scaffoldProjectName}`, path);
+
+const info = (message: string): void => {
+	log(c.bold("\n" + c.blue("i") + c.italic(` ${message}`)));
+};
+const done = (msg: string): void => {
+
+	log(c.greenBright(`\u2713 [Done]: ${msg}`));
+};
+const blueColor = chalk.bold.rgb(38, 45, 255);
+
+function start(): void {
+	log(`
+
+	${blueColor("\n Generator Webpack Scaffold \n")}
+	`);
+
+}
+
+async function question() {
+	info("Details of your scaffold");
+
+	const answers: inquirer.Answers = await inquirer.prompt(QUESTIONS);
+
+	return answers;
+}
+
+function scaffold(answers: inquirer.Answers) {
+	log(`${blueColor("[.....]")} Preparing Your Scaffold Creater Project`);
+
+	const packageJSON = require("./template/package.json");
```

Comment:
Use obj assign to an `options` object and serialize a merged object later

Reply:
Abstract the assigning of props to a util func

Reply:
@evenstensberg still got some doubts here. do you want here `options` object and then merged with `packageJSON` object?

Reply:
something like this ? 
```
let packageJSON = require("./template/package.json");
	const answerOptions = {
		...answers,
		bugs : {
			url : `${answers.github_repo}/issues`,
		},
		homepage : `${answers.github_repo}#readme`,
		name : `webpack-scaffold-${answers.name}`,
		repository : {
			url : `git+${answers.github_repo}`,
		},
	};
	packageJSON = Object.assign({}, packageJSON, answerOptions);
```

Reply:
done with the above changes. Please recommend if this is not what  you wanted @evenstensberg